app/views/_macros/components.nunjucks

```diff
@@ -1,28 +1,30 @@
 {% macro image(options) %}
-  <!--[if gt IE 7]><!-->
-  <img
-    srcset="
-      {% for src in options.srcset %}
-        {% set splitSrc = src | split(' ') %}
-        {% set path = asset_path(splitSrc[0]) %}
-        {% set width = splitSrc[1] %}
-
-        {% if loop.last %}
-          {{ path }} {{ width }}
-        {% else %}
-          {{ path }} {{ width }},
-        {% endif %}
-      {% endfor %}
-    "
-    alt="{{ options.alt }}"
-    data-analytics="image"
-    data-analytics-type="{{ options.analyticsType | default('inline') }}"
-    />
-  <!--<![endif]-->
-  <!--[if gt IE 7]><!--><noscript><!--<![endif]-->
-    {% set fallback = options.srcset[0] | split(' ') %}
-    <img src="{{ asset_path(fallback[0]) }}" alt="{{ options.alt }}" />
-  <!--[if gt IE 7]><!--></noscript><!--<![endif]-->
+  {% if options.srcset %}
```

Comment:
I think the original reason for doing this was so you can't have to pass null or similar to the component if you want to send a value for the last argument but not ones before it.